benchmarks/templates/benchmarks/compare.html

```diff
@@ -4,14 +4,26 @@
     <div class="xlabel-container">
         <select id="xlabel">
             {% for benchmark in benchmarks %}
-                <option value="{{ benchmark.identifier }}">{{ benchmark.short_name }}</option>
+                {% if "average" in benchmark.short_name %}
+                    <option value="{{ benchmark.identifier }}">average</option>
+                {% elif "engineering" in benchmark.short_name %}
+                    <option value="{{ benchmark.identifier }}">engineering</option>
+                {% else %}
+                    <option value="{{ benchmark.identifier }}">{{ benchmark.short_name }}</option>
+                {% endif %}
             {% endfor %}
         </select>
     </div>
     <div class="ylabel-container">
         <select id="ylabel">
             {% for benchmark in benchmarks %}
-                <option value="{{ benchmark.identifier }}">{{ benchmark.short_name }}</option>
+                {% if "average" in benchmark.short_name %}
+                    <option value="{{ benchmark.identifier }}">average</option>
+                {% elif "engineering" in benchmark.short_name %}
+                    <option value="{{ benchmark.identifier }}">engineering</option>
+                {% else %}
+                    <option value="{{ benchmark.identifier }}">{{ benchmark.short_name }}</option>
+                {% endif %}
```

Comment:
This (and the section above) would be more cleanly handled in a template filter, e.g. 
`<option value="{{ benchmark.identifier }}">{{ benchmark.short_name|simplify_domain }}</option>`
where
```python
def simplify_domain(benchmark_name:str) -> str:
    suffixed_benchmarks = ['average', 'engineering']
    for suffixed_name in suffixed_benchmarks:
        if benchmark_name.startswith(f"{suffixed_name}_"):
            return suffixed_name
    return benchmark_name
```